### ISSUE TITLE
I think `.env.local` should be the default location for loading `DATABASE_URL`

### ISSUE THREAD

**Issue by Jinsung-L** [prisma/prisma#15681]
## Problem

`prisma` loads `DATABASE_URL` from `.env` by default but I think it should be `.env.local`.
From my understanding, the convention is that to use `.env` file to store defaults that should be included in the repository and to use `.env.local` file to store secrets.
And I think `DATABASE_URL` is a secret. Something that should not be included in the repository.

> Note: .env, .env.development, and .env.production files should be included in your repository as they define defaults. .env*.local should be added to .gitignore, as those files are intended to be ignored. .env.local is where secrets can be stored.
> https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables

## Suggested solution

Change the default filename for loading environment variables to `.env.local`.

## Alternatives

Add support for loading `.env` file variations: `.env.*`, `.env.*.local`.

## Additional context

https://github.com/prisma/prisma/issues/3865
https://github.com/prisma/prisma/issues/3226

**Comment by janpio** [comment]
You are linking to Next documentation which lists its conventions, here is a counter point from the official `dotenv` package:
https://www.npmjs.com/package/dotenv#:~:text=Should%20I%20commit,values%20between%20environments.

> **Should I commit my .env file?**
> No. We strongly recommend against committing your .env file to version control. It should only include environment-specific values such as database passwords or API keys. Your production database should have a different password than your development database.

But we will consider your argument.

**Comment by isaif** [comment]
> You are linking to Next documentation which lists its conventions, here is a counter point from the official `dotenv` package: https://www.npmjs.com/package/dotenv#:~:text=Should%20I%20commit,values%20between%20environments.
> 
> > **Should I commit my .env file?**
> > No. We strongly recommend against committing your .env file to version control. It should only include environment-specific values such as database passwords or API keys. Your production database should have a different password than your development database.
> 
> But we will consider your argument, as I agree this might not be well defined.

I agree that we should not commit .env to source control but we should have the option to change which .env file prisma use.
For now I am using dotenv-cli to source .env.local file.

**Comment by Jinsung-L** [comment]
> You are linking to Next documentation which lists its conventions, here is a counter point from the official `dotenv` package: https://www.npmjs.com/package/dotenv#:~:text=Should%20I%20commit,values%20between%20environments.
> 
> > **Should I commit my .env file?**
> > No. We strongly recommend against committing your .env file to version control. It should only include environment-specific values such as database passwords or API keys. Your production database should have a different password than your development database.
> 
> But we will consider your argument, as I agree this might not be well defined.

I thought when you have defaults and secrets for a project, it would be more natural to separate them into `.env` and `.env.local`.
Doesn't it often happen when you want to include some default values for the project into source control?

Changing the default env filename might be controversial.
Then how about the alternative solution to load `.env.local` as a fallback?

**Comment by proger-coder** [comment]
Hello. 
Due to widely-distributed usage of 2 separate files  (.env.development + .env.production), **_instead_** of single .env, 
seems to be logical to feat support of such type of .env.* files.
Installing packets to make custom cli commands for start looks like ugly kludge in 2023.
Best regards.
